src/Avalonia.Base/Controls/ResourceDictionary.cs

```diff
@@ -15,6 +15,7 @@ namespace Avalonia.Controls
     /// </summary>
     public class ResourceDictionary : IResourceDictionary
     {
+        private DeferredItem? s_lastDeferredItem;
```

Comment:
This has the `s_` prefix but isn't static. Should it be?(I don't think it should?)

Reply:
Yep, it was static, but then I changed. Forgot to adjust the name as well.